script/helpers.py

```diff
@@ -84,3 +84,13 @@ def clang_check(file_path):
                 file_status = False
 
         return file_status
+
+
+def clang_format(file_path):
+    """Formats the file at file_path"""
+    if CLANG_FORMAT is None:
```

Comment:
Directly print the error (like in https://github.com/schedutron/peloton/blob/2f4825a022ce86f20de2b181411f879657711225/script/helpers.py#L57) and make the function return void.